TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The thing is, this is the only .csproj that works with the CLI's "build3".  So people will start copying it. And then people will make copies of those copies, and the "cruft" will continue to live for a long time.

It would be best to make this as slim as possible now, so the "cruft" is at a minimum.

It also helps us understand what things are "hacks" and what things are "needed".  Having the "cruft" just muddies the file, and doesn't give us a clear vision of the current state.